project/GraalVM.scala

```diff
@@ -100,10 +100,10 @@ object GraalVM {
     // nightly builds: https://github.com/enso-org/enso/pull/12843#issuecomment-2869897463
     // With maximal size relaxed by 30 MB.
     private val windowsX64Release = NativeImageSize(200, 440)
-    private val linuxX64Release   = NativeImageSize(200, 465)
+    private val linuxX64Release   = NativeImageSize(200, 490)
```

Comment:
Why is there an increase in the size of native image?

Reply:
As the name of the commit suggests, and as the comment above the code suggests, there should be some reserve. I believe that https://github.com/enso-org/enso/pull/13144/files#diff-003b80848f25430f0c7c682eaba3bae8f1e12c023c91e1236bc7a72dd1ca474d haven't introduced sufficient reserve. So my NI build was failing - it was just 2MB bigger.